Upstream-Status: Inappropriate [embedded specific]
==================================================
diff --git a/script/service b/script/service
index 08f69bb..14872dc 100755
--- a/script/service
+++ b/script/service
@@ -73,36 +73,7 @@ while [ $# -gt 0 ]; do
        ;;
     *)
        if [ -z "${SERVICE}" -a $# -eq 1 -a "${1}" = "--status-all" ]; then
-          cd ${SERVICEDIR}
-          for SERVICE in * ; do
-            case "${SERVICE}" in
-              functions | halt | killall | single| linuxconf| kudzu)
-                  ;;
-              *)
-                if ! is_ignored_file "${SERVICE}" \
-		    && [ -x "${SERVICEDIR}/${SERVICE}" ]; then
-                        out=$(env -i LANG="$LANG" LANGUAGE="$LANGUAGE" LC_CTYPE="$LC_CTYPE" LC_NUMERIC="$LC_NUMERIC" LC_TIME="$LC_TIME" LC_COLLATE="$LC_COLLATE" LC_MONETARY="$LC_MONETARY" LC_MESSAGES="$LC_MESSAGES" LC_PAPER="$LC_PAPER" LC_NAME="$LC_NAME" LC_ADDRESS="$LC_ADDRESS" LC_TELEPHONE="$LC_TELEPHONE" LC_MEASUREMENT="$LC_MEASUREMENT" LC_IDENTIFICATION="$LC_IDENTIFICATION" LC_ALL="$LC_ALL" PATH="$PATH" TERM="$TERM" "$SERVICEDIR/$SERVICE" status 2>&1)
-                        retval=$?
-                        if echo "$out" | grep -Fiq "usage:"; then
-                          #printf " %s %-60s %s\n" "[?]" "$SERVICE:" "unknown" 1>&2
-                          echo " [ ? ]  $SERVICE" 1>&2
-                          continue
-                        else
-                          if [ "$retval" = "0" -a -n "$out" ]; then
-                            #printf " %s %-60s %s\n" "[+]" "$SERVICE:" "running"
-                            echo " [ + ]  $SERVICE"
-                            continue
-                          else
-                            #printf " %s %-60s %s\n" "[-]" "$SERVICE:" "NOT running"
-                            echo " [ - ]  $SERVICE"
-                            continue
-                          fi
-                        fi
-                  #env -i LANG="$LANG" LANGUAGE="$LANGUAGE" LC_CTYPE="$LC_CTYPE" LC_NUMERIC="$LC_NUMERIC" LC_TIME="$LC_TIME" LC_COLLATE="$LC_COLLATE" LC_MONETARY="$LC_MONETARY" LC_MESSAGES="$LC_MESSAGES" LC_PAPER="$LC_PAPER" LC_NAME="$LC_NAME" LC_ADDRESS="$LC_ADDRESS" LC_TELEPHONE="$LC_TELEPHONE" LC_MEASUREMENT="$LC_MEASUREMENT" LC_IDENTIFICATION="$LC_IDENTIFICATION" LC_ALL="$LC_ALL" PATH="$PATH" TERM="$TERM" "$SERVICEDIR/$SERVICE" status
-                fi
-                ;;
-            esac
-          done
+          echo "Not implemented." >&2
           exit 0
        elif [ $# -eq 2 -a "${2}" = "--full-restart" ]; then
           SERVICE="${1}"
